cmd/frontend/internal/app/jscontext/jscontext.go

```diff
@@ -125,6 +126,11 @@ func NewJSContextFromRequest(req *http.Request, db database.DB) JSContext {
 	// Auth providers
 	var authProviders []authProviderInfo
 	for _, p := range providers.Providers() {
+		if p.Config().Github != nil {
+			if p.Config().Github.Hidden {
+				continue
+			}
+		}
```

Comment:
```suggestion
		if p.Config().Github != nil && p.Config().Github.Hidden {
			continue
		}
```

This is safe because [Go uses short circuit evaluations](https://stackoverflow.com/a/53722705).